docs/quick_run.md

```diff
@@ -46,6 +46,8 @@ Examples:
 python demo/demo_mot.py configs/mot/deepsort/sort_faster-rcnn_fpn_4e_mot17-private.py --input demo/demo.mp4 --output mot.mp4
 ```
 
+Note: The multiple object tracking model must use the private detectors.
```

Comment:
Change it to 'When running `demo_mot.py`, we suggest you use config containing `private`, since `private` means the MOT method doesn't need external detections.'
please also add these sentences into the readme of sort and tracktor method.